### PR TITLE
Fix Stark Affiliate section

### LINKED ISSUE #20: New stark affiliate program section
## Description 📹

Loom: [https://www.loom.com/share/036e8ad7ead349b188733c768ee23479](https://www.loom.com/share/036e8ad7ead349b188733c768ee23479)

We've refactored the Stark affiliate program to have something more clear & straightforward. This section allows users to copy their referral link for sharing.

Old section: [https://affiliate.starknet.id/](https://affiliate.starknet.id/)  
image

New section:  
[https://www.figma.com/design/S1UKYgWewNqNHZFAaBUilG/%F0%9F%8F%9D%EF%B8%8F-Starknet-ID?node-id=3555-6782&t=x8xvLBhT37BVQyXF-1](https://www.figma.com/design/S1UKYgWewNqNHZFAaBUilG/%F0%9F%8F%9D%EF%B8%8F-Starknet-ID?node-id=3555-6782&t=x8xvLBhT37BVQyXF-1)  
image

## Proposed Actions 🛠️

Here’s a checklist of actions to follow for resolving this issue:

1. **Fork and Create Branch**:  
   Fork the repository and create a new branch using the issue number:
   ```bash
   git checkout -b fix-[issue-number]
   ```

2. **Run Tests and Commit Changes**:  
   Make sure your changes don't break existing functionality and commit with a clear message:
   ```bash
   git commit -m "Fix: [Short description of the fix]"
   ```

## Required 📋

To keep our workflow smooth, please make sure you follow these guidelines:

- **Assignment**: Don't create a pull request if you weren’t assigned to this issue.
- **Timeframe**: Complete the task within **3 business days**.
- **Closing the Issue**: In your PR description, close the issue by writing `Close #[issue_id]`.
- **Review Process**:
  - Once you've submitted your PR, change the label to **"ready for review"**.
  - If changes are requested, address them and then update the label back to **"ready for review"** once done.
- **Testing**: Test your PR locally before pushing, and verify that tests and build are working after pushing.

Thank you for your contribution 🙏

⚠️ WARNING: Failure to follow the requirements above may result in being added to the OnlyDust blacklist, affecting your ability to receive future rewards.

### PR DESCRIPTION
**Pull request by wheval**
Closes #20 

quest

### CODE REVIEW

File: components/UI/affiliateLink.tsx [hunk 1]
```diff
@@ -12,7 +10,7 @@ interface AffiliateLinkProps {
 
 export const AffiliateLink: FC<AffiliateLinkProps> = ({ link, domain }) => {
   const [copied, setCopied] = useState(false);
-  const linkToShow = `${process.env.NEXT_PUBLIC_APP_LINK}/${domain}`;
+  const linkToShow = `https://app.starknet.id/${domain}`;
```
Comment (fricoben) [inline components/UI/affiliateLink.tsx]: keep it :)

**Review by fricoben — CHANGES_REQUESTED** [review]
change this to a confirm with a little green tick

Capture d’écran 2024-11-23 à 21 40 59

File: components/UI/affiliateLink.tsx (hunk 1 above)
Comment (wheval) [reply components/UI/affiliateLink.tsx]: okay!

**Comment by wheval** [comment]
@fricoben. Please review

**Review by fricoben — APPROVED** [review]
lgtm

**Pull request merged**